lib/tls.js

```diff
@@ -15,9 +15,9 @@ exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
 exports.DEFAULT_CIPHERS =
     // TLS 1.2
-    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' +
+    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:' +
```

Comment:
I may have misread your other comment.  So you want to drop AES128-GCM-SHA256 in favor of AES256-GCM-SHA384 and a long list of other ciphers?  I don't disagree per se but recall that the last time we made the cipher list bigger, it broke a number of load balancers and SSL terminators that couldn't handle handshakes > 64 bytes.

I don't think that's a reason to reject such changes but if you land them, it warrants a minor version bump and a mention in the release notes.